src/main/java/games/strategy/util/TimeManager.java

```diff
@@ -1,20 +1,54 @@
 package games.strategy.util;
 
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.SimpleTimeZone;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.FormatStyle;
 
 public class TimeManager {
   /**
-   * Replacement for Date.toGMTString();
+   * Prints an {@link Instant} localized, with all details
    *
-   * @param date The Date being returned as String
+   * @param instant The {@link Instant} being returned as String
    * @return formatted GMT Date String
    */
-  public static String getGMTString(final Date date) {
-    final DateFormat dateFormat = new SimpleDateFormat("dd MMM yyyy HH:mm:ss z");
-    dateFormat.setTimeZone(new SimpleTimeZone(0, "GMT"));
-    return dateFormat.format(date);
+  public static String getFullUTCString(final Instant instant) {
```

Comment:
There was an existing Checkstyle abbreviation violation here, but since you renamed the method in this PR... can you convert `UTC` to `Utc` please? :smile: